### DISCUSSION TITLE
Lyrics in miniplayer similar to Apple Music app on windows

### DISCUSSION THREAD

**Discussion by agrmohit** [ciderapp/Cider-2#386] [Feature Requests]
Currently Cider 2 miniplayer only has basic controls with no lyrics support
![image](https://github.com/ciderapp/Cider-2/assets/32242258/37b95a16-28d9-4b77-8df1-0da83a6c2b2c)

It would be nice to be able to optionally expand the miniplayer such that it also displays lyrics like the Apple Music app.
![image](https://github.com/ciderapp/Cider-2/assets/32242258/e5e81b85-0584-47dd-aec8-8f1b58cb90d7)

**Comment by tscharich** [comment]
I would like to suggest the other displays of the mini-player including the one with your queue and the smaller mini-player.

![image](https://github.com/ciderapp/Cider-2/assets/13457461/dcaf6771-9013-4771-9b76-33c54da1e07d)
![image](https://github.com/ciderapp/Cider-2/assets/13457461/12283d2f-db20-4263-8eb1-5978651c0585)
